test/framework/e2e/elasticsearch.go

```diff
@@ -93,7 +93,63 @@ type ElasticLogStore struct {
 }
 
 func (es *ElasticLogStore) ApplicationLogs(timeToWait time.Duration) (types.Logs, error) {
-	panic("Method not implemented")
+	app_index := "app-000001"
+	var allLogs types.Logs
+	var logDoc types.AllLog
+	err := wait.PollImmediate(defaultRetryInterval, timeToWait, func() (done bool, err error) {
+		cmd := fmt.Sprintf("--query=%s/_search?pretty=true", app_index)
+		options := metav1.ListOptions{
+			LabelSelector: "component=elasticsearch",
+		}
+		pods, err := es.Framework.KubeClient.CoreV1().Pods(constants.OpenshiftNS).List(context.TODO(), options)
+		if err != nil {
+			return false, err
+		}
+		if len(pods.Items) == 0 {
+			return false, errors.New("No pods found for elasticsearch")
+		}
+		clolog.V(3).Info("Pod ", "PodName", pods.Items[0].Name)
+
+		result, err := es.Framework.PodExec(constants.OpenshiftNS, pods.Items[0].Name, "elasticsearch", []string{"es_util", cmd})
+		if err != nil {
+			return false, err
+		}
+		if result != "" && err == nil {
+			var elasticResult map[string]interface{}
+			err = json.Unmarshal([]byte(result), &elasticResult)
+			if err == nil {
+				if elasticResult["timed_out"] == false {
+					resultHits := elasticResult["hits"].(map[string]interface{})
```

Comment:
This same/similar code lives in test/functional/elasticsearch.go.  Please find commonality that can be refactored into test/helpers/elasticsearch/elasticsearch.go